convene-web/spec/requests/spaces/invitations_request_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe "/spaces/:space_id/invitations", type: :request do
+  it "doesn't expsose" do
```

Comment:
Maybe blocking: This `it` statement probably wants a better wording. Perhaps `it "creates an invitation to the Space"`